cfme/networks/views.py

```diff
@@ -374,6 +374,18 @@ class my_filters(Accordion):  # noqa
         tree = ManageIQTree()
 
 
+class SubnetNetworkPortView(NetworkPortView):
+    """ Represents a Subnet network with port view"""
+    title = Text('//div[@id="main-content"]//h1')
+
+    @property
+    def is_displayed(self):
+        return (super(BaseLoggedInPage, self).is_displayed and
```

Comment:
I think a better call here is to not use `super` and directly call `self.logged_in_as_current_user`, as this will account for the context (UI, SSUI) and the active appliance user.

Reply:
Pushing a commit with this change, and a rebase due to the import order sorting PR #8487 being merged.